packages/vehicle-lifecycle-network/test/vda.js

```diff
@@ -124,4 +124,32 @@ describe('Vehicle Lifecycle Network', function() {
                 });
         });
     });
-});
\ No newline at end of file
+
+    describe('ScrapAllVehiclesByColour', function() {
+        it('should select vehicles by colour and change vehicles status to SCRAPPED', function() {
+            /* vehicle with beige colour
+               and id 123456789 resides
+               in reposritory
+            */
+            var vehicleToScrap = 'Beige';
+            var assetRegistry;
+            var vehicleId = '123456789';
+
+            var scrapVehicle = factory.newTransaction(NS_D, 'ScrapAllVehiclesByColour');
```

Comment:
exceptionally pedantic request, but could this be 'scrapVehicleByColourTxn' as I got a little confused reading the test code.

Reply:
I have made this change but this require model file vda.cto to change which is in composer-sample-models repository. I have updated the file. But since it is a separate repository I have to raise a separate PR 1500 #36 for it.